src/components/__tests__/SkeletonButton.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { renderIntoDocument, render, fireEvent } from 'react-testing-library'
+import SkeletonButton from '../SkeletonButton'
+
+it('renders without crashing', () => {
+  const { container } = render(
+    <SkeletonButton>Test SkeletonButton</SkeletonButton>,
+  )
+  expect(container.firstChild).toMatchSnapshot()
+})
+
+it('renders with custom width', () => {
+  const { container } = render(
+    <SkeletonButton width={360}>Custom Width SkeletonButton</SkeletonButton>,
+  )
+  expect(container.firstChild).toMatchSnapshot()
+})
```

Comment:
Should we test with a percentage width too?

Reply:
Added.